labs/blockchain-analyzer.md

```diff
@@ -0,0 +1,23 @@
+# Lab Name
+Blockchain Analyzer
+
+# Short Description
+The aim of this lab is to create a tool for visualizing and analyzing Hyperledger Fabric ledger from both operational and data-oriented aspects. It provides dashboards similar to Hyperledger Explorer, but it is applicable for data queries (e.g. medicine provenance, supply chain) as well.
+
+The user can inspect the data of multiple peers and multiple channels at the same time.
+
+A further goal of this project is to be suitable for any blockchain solution.
+
+# Scope of Lab
+
+Internship project ([Analyzing Hyperledger Fabric Ledger, Transactions and Logs using Elasticsearch and Kibana](https://wiki.hyperledger.org/display/INTERN/Analyzing+Hyperledger+Fabric+Ledger%2C+Transactions%2C+and+Logs+using+Elasticsearch+and+Kibana))
+
+# Initial Committers
+- https://github.com/salmanbaset
+- https://github.com/balazsprehoda
+
+# Sponsor
+- https://github.com/christo4ferris - Chair of Hyperledger TSC
+
+# Pre-existing repository
+- https://github.com/balazsprehoda/hyperledger-elastic
```

Comment:
It appears that this repository does not use the `-s` option for its commits and does not have DCO sign offs that will be required for Hyperledger Labs repositories. When this gets brought over to Labs, it will not be able to be directly transferred, as is.

Reply:
I squashed all commits on master branch with DCO, is that satisfactory?